docs/tutorials/integrations/https-nginx.md

```diff
@@ -32,13 +35,50 @@ import LetsEncrypt from './tab-nginx/LetsEncrypt.md';
   </TabItem>
 </Tabs>
 
-## Next Steps
+## Additional Tips
 
-After setting up HTTPS, access Open WebUI securely at:
+### Increase Upload Size
 
-- [https://localhost](https://localhost)
+To allow users to upload larger files to OpenWebUI, increase the default upload size limit in your Nginx configuration:
 
-Ensure that your DNS records are correctly configured if you're using a domain name. For production environments, it's recommended to use Let's Encrypt for trusted SSL certificates.
+```nginx
+client_max_body_size 20M;  # Increase upload size limit to 20MB
+```
 
----
+Add this directive inside your `server` block, above the `location /` block.
+
+## Customize Splash Screen and Favicon
```

Comment:
Let's leave a note here for sponsorship here.

Reply:
something like this?...

````
### Important Note on Sponsorship and Attribution

While Nginx can be used to rebrand the Open WebUI (e.g., replacing the splash screen or favicon), it’s important to acknowledge that this open-source project relies on the generous support of its sponsors. Rebranding without attribution can undermine the efforts of those funding the project’s ongoing development.

**Recommendation:**  
If rebranding is implemented using the methods described here, please provide some attribution back to the Open WebUI project (e.g., by mentioning the project in your modified version). This ensures fairness and continued support for the open-source community.
````